src/components/sidebar/Sidebar.tsx

```diff
@@ -0,0 +1,51 @@
+import { Button, Container, Form, Col, Row} from 'react-bootstrap';
+
+
+function Sidebar() {
+  return (
+    <Container className='Sidebar-container'>
+        <Col>
+            <Form>
+                <Row>
+                    <Form.Group className="mb-3" controlId="formBasicEmail">
+                        <Form.Label>Parameter IDs</Form.Label>
```

Comment:
Seems like you're also adding a new feature in this PR? Can you put it in the PR description or split it out?

Reply:
I think something snuck in when I was fixing merge conflicts. I meant to only change the one file GranuelTable.tsx